bin/logs.js

```diff
@@ -32,6 +32,12 @@ module.exports = Cli.createCommand('logs', 'Streaming, real-time logs', {
             'default': 'default',
             type: 'string',
         },
+        lines: {
```

Comment:
I wasn't sure so I just stole the name that `journalctl` uses (though systemd is hardly a paragon of usability...)

`previous` or `history` might work?